lib/clients/transports/request.js

```diff
@@ -34,6 +34,10 @@ var getRequestTransportArgs = function getReqestTransportArgs(args) {
     transportArgs.form = args.data;
   }
 
+  if (has(args, 'body')) {
```

Comment:
There was no test coverage around this file that I could fine. Is it cool to add this?